keyboards/scrabblepad/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "scrabblepad.h": "c"
+    }
+}
```

Comment:
Would you mind removing this or adding it to the gitignore?

Reply:
Sure thing!